### ISSUE TITLE
Incorrect shrieking legiana data: shock/pitfall traps should be TRUE

### ISSUE THREAD

**Issue by DuckBoss** [gatheringhallstudios/MHWorldData#47]
https://github.com/gatheringhallstudios/MHWorldData/blob/master/source_data/monsters/monster_base.csv#L67

The shrieking legiana can be caught by shock/pitfall traps I believe,  and the  source data in the repository says it can't.

**Comment by CarlosFdez** [comment]
Good catch. For those monsters I believe the reason I didn't was because I wasn't sure, but I don't think there's any non-elder monster that resists any trap, except *maybe* a charged zinogre? Especially since there's no Gypceros.

**Comment by DeveloperPaul123** [comment]
Correct, charged Zinogre cannot be caught by shock traps. Also, Rajang cannot be trapped when in his "super enraged" state. There are similar conditions with Furious Rajang as well.

**Comment by DuckBoss** [comment]
I was going through the database, and I noticed that the Gold Rathian also has the same issue as the Shrieking Legiana where the shock/pitfall traps are not set to TRUE.

**Comment by TrenchTemplar** [comment]
Commenting onto an issue that's quite old but I just came across this thread.
Some additional notes on monsters that change their ability to be trapped based on states that are not entirely correct as shown currently: 

Nargacuga
Normal State:
CANNOT be Pitfall Trapped
CAN be shock trapped

Enraged State:
BOTH traps are effective

Zinogre
Normal State:
BOTH traps are effective *note a shock trap will send him directly into his "charged state" as soon as the trap ends

Charged State:
CANNOT be shock trapped
CAN be pitfall trapped

Rajang + Furious Rajang
Normal State: 
CANNOT be pitfall trapped (he jumps on them and breaks them)
CAN be shock trapped

Enraged State: 
CAN be pitfall trapped
CANNOT be shock trapped (he picks it up and crushes it)

*edit clarification
